README.md

```diff
@@ -38,6 +31,24 @@ To build Ghaf documentation, use:
 See the documentation overview under [README-docs.md](./docs/README-docs.md).
 
 
+## Other Project Repositories
+
+Other repositories that are a part of the Ghaf project:
+
+* <https://github.com/tiiuae/sbomnix>: a utility that generates SBOMs given Nix derivations or out paths
+* <https://github.com/tiiuae/ci-public>, <https://github.com/tiiuae/ci-test-automation>: CI/CD related files
```

Comment:
ghaf-infra project also could be included?

Reply:
Done. Now it looks like this:

Other repositories that are a part of the Ghaf project:
- sbomnix: a utility that generates SBOMs given Nix derivations or out paths
- **ghaf-infra**, ci-public, ci-test-automation: CI/CD related files